pages/base.py

```diff
@@ -19,6 +19,10 @@ class Base(Page):
 
     _pending_approval_locator = (By.ID, 'pending-approval')
     _account_created_successfully_locator = (By.CSS_SELECTOR, 'div.alert:nth-child(2)')
+    _login_with_email_button_locator = (By.CSS_SELECTOR, '.auth0-lock-passwordless-button.auth0-lock-passwordless-big-button')
+    _email_input = (By.CSS_SELECTOR, '.auth0-lock-passwordless-pane>div>div>input')
+    _send_email_button_locator = (By.CSS_SELECTOR, '.auth0-lock-passwordless-submit')
+    _email_sent_successful_message_locator = (By.CSS_SELECTOR, '.auth0-lock-passwordless-confirmation>p')
```

Comment:
These locators are not used, and duplicates of those in the AuthZero page object.